templates/terraform/modules/cognito/variables.tf

```diff
@@ -0,0 +1,7 @@
+variable "user_pool" {
+  description = "AWS Cognito pool name"
+} 
+variable "hostname" {
+  default = "{{ .Config.Frontend.Hostname }}"
```

Comment:
This shouldn't be a default, it should be passed in from where the module is being used.